src/user_actions_handler.py

```diff
@@ -11,12 +12,23 @@
 except FileNotFoundError:
     book = AddressBook()
 
+ 
+def input_error(func):
```

Comment:
no need to create separate input_error. Please use one that already in the code (file: error_handler.py), but extend it according to your errors.